code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -126,10 +126,10 @@
 		return src.close()
 	return src.open()
 
-// this should probably use dump_contents()
 /obj/structure/closet/ex_act(severity, target)
-	contents_explosion(severity, target)

```

Comment:
I'm pretty sure removing this means mobs inside closets are protected from any explosioneven if they are right next to the epicenter.